parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -429,6 +432,17 @@ should_skip_nvidia_drivers() {
     echo "$should_skip"
 }
 
+should_disable_kubelet_serving_certificate_rotation() {
+    set -x
+    body=$(curl -fsSL -H "Metadata: true" --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01")
```

Comment:
don't we have a helper function to query the metadata service ?

Reply:
let me double check, though I don't think we do

Reply:
I decided to copy the format used for the skip gpu driver function, I think we should probably align on a style guide eventually...

Reply:
double checked we don't, might be worth a follow-up PR to add a helper